edgexfoundry/device-usb-camera
# feat: implement get/set pixel format commands.

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

1. Build docker image using `make docker` and deploy all edgex core services along with the usb service.
1. Executing get `PixelFormat` command for a valid camera with a valid path index should give valid result with height, width, pixelFormat and other parameters.
2. Invalid camera or path index (not video streaming path) should give error.
3. Executing set `PixelFormat` command with valid parameters (height, width and pixelFormat) should be successful (200 status code). Postman collection can be referred for example. Execute get command to validate new parameters have been set.
4. Invalid values should give errors.

## New Dependency Instructions (If applicable)
